third_party/mbedtls/repo/library/ctr_drbg.c

```diff
@@ -178,7 +178,7 @@ static int block_cipher_df( unsigned char *output,
     buf_len = MBEDTLS_CTR_DRBG_BLOCKSIZE + 8 + data_len + 1;
 
     for( i = 0; i < MBEDTLS_CTR_DRBG_KEYSIZE; i++ )
-        key[i] = i;
+        key[i] = (unsigned char)i;
```

Comment:
Changes like this should be contributed upstream. We generally want to keep all sources in `third_party` as pristine releases from those projects, but exceptions are made on a temporary basis assuming the changes have been contributed upstream and `README.google` has been updated to indicate what changes have been made.